plugins/module_utils/mysql.py

```diff
@@ -37,6 +38,46 @@
 from ansible_collections.community.mysql.plugins.module_utils.database import mysql_quote_identifier
 
 
+def get_connector_name(connector):
+    """ (class) -> str
+    Return the name of the connector (pymysql or mysqlclient (MySQLdb))
+    or 'Unknown' if not pymysql or MySQLdb. When adding a
+    connector here, also modify get_connector_version.
+    """
+    if connector is None or not hasattr(connector, '__name__'):
+        return 'Unknown'
+
+    if connector.__name__ not in ['pymysql', 'MySQLdb']:
+        return 'Unknown'
```

Comment:
```suggestion
    if connector.__name__ not in ['pymysql', 'MySQLdb']:
        return 'Unknown'
```
should we maybe return anything it returns?

Reply:
e.g. if in future they, say, decide to return `mysqlclient` or whatever it'll be `Unknown`